index.js

```diff
@@ -0,0 +1,7 @@
+document.getElementById("myCanvas");
+
+function bouncingballs(parameter1, parameter2, parameter3) {
```

Comment:
Few comments here:
- In recent years (with es6) the new format for a function is normally "const bouncingballs = () => {...};, see https://www.w3schools.com/js/js_es6.asp#mark_arrow.
- mind the naming of the function depending on what its meant to do, "bouncingballs" is not so clear as to what the task of it is, also mind the camel case eg bouncingBalls is preferred. 
- parameter names should be more descriptive and intuitive for the reader

Reply:
Good thing to know! I had totally missed this new way of writing functions. 

Regarding the parameter names, it's not done yet in this stage so it was just a placeholder at the time.